addOns/encoder/encoder.gradle.kts

```diff
@@ -12,3 +12,11 @@ zapAddOn {
         url.set("https://www.zaproxy.org/docs/desktop/addons/encode-decode-hash/")
     }
 }
+
+crowdin {
+    configuration {
+        val resourcesPath = "org/zaproxy/addon/${zapAddOn.addOnId.get()}/resources/"
+        tokens.put("%messagesPath%", resourcesPath)
+        tokens.put("%helpPath%", resourcesPath)
```

Comment:
So that I might know for the future (hopefully I can remember) what's the difference here vs exim?

Reply:
Actually we seem to have a couple variations, httpsinfo being the most different I guess (probably because it changed name/id at some point). Is it easy to boil down the differences? Is it worth noting them in some docs somewhere?

Reply:
Yes, too many variations. It boils down to where the resource files are placed basically. If you provide the wrong path you get an error telling you the directories don't exist, so not sure it needs to be explicitly mentioned somewhere.

For the previous case:
```
encoder/src/main/javahelp/org/zaproxy/addon/encoder/resources/help/
exim/src/main/javahelp/help/
```

Reply:
Thanks!